examples/olsconfig.yaml

```diff
@@ -11,6 +11,8 @@ LLMProviders:
     models:
       - name: gpt-4-1106-preview
 OLSConfig:
+  product_docs_persist_dir: "./vector-db/ocp-product-docs"
+  summary_docs_persist_dir: "./vector-db/summary-docs"
```

Comment:
This should be nested under a new config structure for "reference_content" or something along those lines.  We are likely to going to end up with a handful of config items related to RAG stuff and we aren't going to want them to all be a bunch of top level fields.

Reply:
@bparees : What about rag_content, or rag_config or just rag?

Reply:
i don't really want "rag" to appear in our config spec field names.  it's not sufficiently self-explanatory.